boards/DCM/Src/main.c

```diff
@@ -92,6 +94,7 @@ struct BrakeLight *       brake_light;
 struct Buzzer *           buzzer;
 struct ErrorTable *       error_table;
 struct Clock *            clock;
+struct Imu *              imu;
```

Comment:
nit; As previously discussed, the Imu should be grouped with the other objects

Reply:
fixed